source/tests/pt_expt/test_multitask.py

```
@@ -0,0 +1,2099 @@
+        mt_config["model"], shared_links = preprocess_shared_params(mt_config["model"])
+        mt_config = update_deepmd_input(mt_config, warning=False)
+        mt_config = normalize(mt_config, multi_task=True)
+
+        mt_trainer = get_trainer(deepcopy(mt_config), shared_links=shared_links)
+        mt_desc = mt_trainer.wrapper.model["model_1"].atomic_model.descriptor
+        mt_fit_1 = mt_trainer.wrapper.model["model_1"].atomic_model.fitting_net
+        mt_fit_2 = mt_trainer.wrapper.model["model_2"].atomic_model.fitting_net
+
+        # Verify descriptor params are aliased (share_params)
+        mt_desc_2 = mt_trainer.wrapper.model["model_2"].atomic_model.descriptor
+        for (n1, p1), (n2, p2) in zip(
```

Ruff (0.15.10)

[warning] Loop control variable `n2` not used within loop body

Rename unused `n2` to `_n2`

(B007)